python/sdk/tests/test_detect.py

```diff
@@ -62,8 +62,10 @@ def test_update_endpoint(self, unwrap):
         assert not diffs, json.dumps(diffs, indent=2)
 
     def test_schedule_threat(self, unwrap):
-        res = unwrap(self.detect.schedule)(self.detect, [dict(threat_id=pytest.threat_id, run_code=RunCode.DAILY)])
-        assert res['id'] == pytest.threat_id
+        res = unwrap(self.detect.schedule)(self.detect, [dict(threat_id=pytest.threat_id, code=RunCode.DAILY.name)])
+        assert 1 == len(res), json.dumps(res, indent=2)
+        diffs = check_dict_items(dict(threat=pytest.threat_id, run_code=RunCode.DAILY.value, tag=None), res[0])
```

Comment:
so the request uses `code` but the response has `run_code`?

Reply:
yeah, that's not great is it...

Reply:
if not too late I think we should fix that